projects/pgai/pgai/vectorizer/embeddings.py

```diff
@@ -414,3 +464,85 @@ async def _encode(self, documents: list[str]) -> list[list[int]]:
     @cached_property
     def _encoder(self) -> tiktoken.Encoding:
         return tiktoken.encoding_for_model(self.model)
+
+
+@dataclass
+class OllamaOptions:
```

Comment:
I added this for completeness, these are the runtime parameters that ollama can take. We don't expose them in the SQL API yet, so I'm tempted to remove them.

Reply:
I *think these all apply to generation rather than embedding